src/utils.js

```diff
@@ -277,41 +277,41 @@ go.utils = {
         return go.utils
             .service_api_call("identities", "post", null, payload, 'identities/', im)
             .then(function(json_post_response) {
-                var contact_created = json_post_response.data;
-                // Return the contact
-                return contact_created;
+                var identity_created = json_post_response.data;
+                // Return the identity
+                return identity_created;
```

Comment:
just looking at the function, let's not create the var for no reason but rather just return json_post_response.data.